### PR TITLE
Collapsing sidebar

### LINKED ISSUE #84: Hide/Collapse the 'New Layer'
I love this project and am trying to use it with some pre-built layers.

Visually, the left hand component where one can define layers takes a lot of real estate on the screen and it would be great if it could be collapsed.  The map itself as a layer icon that expands allowing the user to select which layers to show, which is great.

This way the user can focus on the map and layer data.

### PR DESCRIPTION
**Pull request by nikita03565**
closes #84 
This might be not perfect solution but i don't like css and css doesn't like me.

image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,90 +3,101 @@ import download from "downloadjs";
...
-				} catch (err) {
-					// TODO: Build error UI
-					console.log('Failed to load and parse data from file', err);
-				}
-			};
-			fileReader.readAsText(file);
-		};
-		e.preventDefault();
-	};
+  const defaultMapOffset = 30;
```
Comment (stellasia) [inline src/App.js:73]: Are you sure these numbers work on different screen size?

**Comment by stellasia** [comment]
Nice! I think the way you are doing it is fine. 

I remember having had similar issue with map only fitting the space on window resize, will try to remember the root cause and the solution because it was quite a long time ago :)

File: src/App.js (hunk 1 above)
Comment (nikita03565) [reply src/App.js:73]: not much but 
image

Comment (nikita03565) [reply src/App.js:73]: i'll try to come up with something better

**Comment by nikita03565** [comment]
> Nice! I think the way you are doing it is fine.
> 
> I remember having had similar issue with map only fitting the space on window resize, will try to remember the root cause and the solution because it was quite a long time ago :)

nvm, i found the solution https://stackoverflow.com/questions/24412325/resizing-a-leaflet-map-on-container-resize

File: src/App.js (hunk 1 above)
Comment (nikita03565) [reply src/App.js:83]: Super small window:
image

Fullscreen window at 1920*1080 display
image

cant test 2k/4k resolutions

Comment (nikita03565) [reply src/App.js:87]: without fixing width to 60px sidebar'd been hiding a bit too much
image

**Pull request merged**